src/LibraryUtilities.ts

```diff
@@ -667,12 +701,17 @@ export function setItemStateRecursive(items: ItemData | ItemData[], visible: boo
             item.expanded = false;
         }
 
+        //All groups defined in layoutspec.json should be expanded by default.
+        if (item.itemType === "coregroup") {
+            item.expanded = true;
+        }
+
         setItemStateRecursive(item.childItems, visible, expanded);
     }
 }
 
 export function search(text: string, item: ItemData) {
-    if (item.itemType !== "group") {
+    if (item.itemType !== "group" && item.itemType !== "coregroup") {
```

Comment:
should we also ignore class during search?

Reply:
I don't think so. the idea is to change to class and use the search